fossunited/hooks.py

```diff
@@ -100,3 +100,11 @@
 # Migration
 # ----------------
 after_migrate = "fossunited.migrate.after_migrate"
+
+# Scheduler Events
+# ----------------
+scheduler_events = {
+    "daily_long": [
```

Comment:
Does this run at the beginning of the day e.g. 00:01 hours or does it run at the end of the day e.g. 23:59 hours? Because this isn't clear from the docs

looking at the `conclude_events` function, it seems to be run at 00:01 hours, is that correct? and the events from yesterday/earlier will be updated today.

Reply:
I think it runs at sharp 00:00

Deducing it from frappe's tests: https://github.com/frappe/frappe/blob/develop/frappe/tests/test_scheduler.py